exercises/target/example.js

```diff
@@ -0,0 +1,17 @@
+export default function solve(x, y) {
+  // Define as numbers
+  x = Number(x);
+  y = Number(y);
+
+  // Check for NaN
+  if (x !== x || y !== y) return null;
```

Comment:
Let's be more semantic _and_ validate as early as possible, before the `Number()` calls:

```javascript
if (isNaN(x) || isNaN(y)) return null;
```